html/php-pre-challenge2/index.php

```diff
@@ -2,9 +2,23 @@
 $array = explode(',', $_GET['array']);
 
 // 修正はここから
-for ($i = 0; $i < count($array); $i++) {
+// バブルソート
+// 参考：https://mikaduki.info/webprogram/php/1120/
 
+//並べ替え自体を行うセット回数
+for ($i = 0; $i < count($array); $i++)
+{
+    $next = count($array) - $i;
+    //並べ替えセット一回で行う内部の並べ替え回数
+    for ($j=1; $j < $next; $j++) {
```

Comment:
内側のループでもcountの計算結果が使えます

```suggestion
    $next = $length - $i;
    //並べ替えセット一回で行う内部の並べ替え回数
    for ($j = 1; $j < $next; $j++) {
```